docs/developer_guide/deployment.md

```diff
@@ -31,17 +31,22 @@ scheduler and admission controller. It is primarily intended for developers.
 ## Build docker image
 
 Under project root of the `yunikorn-k8shim`, run the command to build an image using the map for the configuration:
+
+Build docker image can be triggered by running following command.
```

Comment:
Let's combine with the previous sentence (i.e. ... run the command to build YuniKorn Docker images: